Mathlib/Algebra/Module/LinearMap/Basic.lean

```diff
@@ -995,6 +1024,18 @@ instance module : Module S (M →ₛₗ[σ₁₂] M₂) where
 instance [NoZeroSMulDivisors S M₂] : NoZeroSMulDivisors S (M →ₛₗ[σ₁₂] M₂) :=
   coe_injective.noZeroSMulDivisors _ rfl coe_smul
 
+variable {S : Type*} [Semiring S] [Module S M]
+
+instance : Module Sᵈᵐᵃ (M →+ M₂) where
+  add_smul s s' f := AddMonoidHom.ext fun m ↦ by
+    simp_rw [AddMonoidHom.add_apply, DomMulAct.smul_addMonoidHom_apply, ← map_add, ← add_smul]; rfl
+  zero_smul _ := AddMonoidHom.ext fun _ ↦ by
+    erw [DomMulAct.smul_addMonoidHom_apply, zero_smul, map_zero]; rfl
```

Comment:
This seems like it's in the wrong place; can't you put it near the other module instance on additive maps?